### ISSUE TITLE
Packaging for openSUSE: Check test_get_files__return_files_in_directory__success fails

### ISSUE THREAD

**Issue by johanneskastl** [wayfair-incubator/kdave#64]
## Description

I am trying to package this for openSUSE. One of the checks fails.

## Expected Behaviour

Checks should succeed.

## Actual Behaviour

```
[   15s] =================================== FAILURES ===================================
[   15s] ______________ test_get_files__return_files_in_directory__success ______________
[   15s] 
[   15s]     def test_get_files__return_files_in_directory__success():
[   15s] >       assert app.get_files("tests/fixtures/dir") == [
[   15s]             "tests/fixtures/dir/file-a.yaml",
[   15s]             "tests/fixtures/dir/file-b.yaml",
[   15s]             "tests/fixtures/dir/file-c.yaml",
[   15s]         ]
[   15s] E       AssertionError: assert ['tests/fixtures/dir/file-b.yaml', 'tests/fixtures/dir/file-a.yaml', 'tests/fixtures/dir/file-c.yaml'] == ['tests/fixtures/dir/file-a.yaml', 'tests/fixtures/dir/file-b.yaml', 'tests/fixtures/dir/file-c.yaml']
[   15s] E         At index 0 diff: 'tests/fixtures/dir/file-b.yaml' != 'tests/fixtures/dir/file-a.yaml'
[   15s] E         Full diff:
[   15s] E           [
[   15s] E         +  'tests/fixtures/dir/file-b.yaml',
[   15s] E            'tests/fixtures/dir/file-a.yaml',
[   15s] E         -  'tests/fixtures/dir/file-b.yaml',
[   15s] E            'tests/fixtures/dir/file-c.yaml',
[   15s] E           ]
[   15s] 
[   15s] tests/test_app.py:330: AssertionError
[   15s] =============================== warnings summary ===============================
[   15s] tests/test_app.py: 35 warnings
[   15s]   /home/abuild/rpmbuild/BUILD/kdave-0.2.2/exporter/app.py:238: PendingDeprecationWarning: Function 'semver.compare' is deprecated. Deprecated since version 3.0.0.  Still under investigation, see #258. Use the respective 'semver.Version.compare' instead.
[   15s]     if semver.compare(
[   15s] 
[   15s] tests/test_app.py: 14 warnings
[   15s]   /home/abuild/rpmbuild/BUILD/kdave-0.2.2/exporter/app.py:267: DeprecationWarning: Function 'semver.parse_version_info' is deprecated. Deprecated since version 2.10.0.  This function will be removed in semver 3. Use 'semver.version.Version.parse' instead.
[   15s]     ver = semver.parse_version_info(parse_semver(version))
[   15s] 
[   15s] -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
[   15s] =========================== short test summary info ============================
[   15s] FAILED tests/test_app.py::test_get_files__return_files_in_directory__success

```

## Affected Version

0.2.2 taken from the GitHub release page

## Checklist

- [X] I have read the [contributing guidelines](https://github.com/wayfair-incubator/oss-template/blob/main/CONTRIBUTING.md)
- [X] I have verified this does not duplicate an existing issue

**Comment by johanneskastl** [comment]
Hmmm, seems like this is solved in 0.2.3 (GitHub tarball).

**Comment by johanneskastl** [comment]
The following message is still present, but does not lead to a failed build:

```
[   88s] =============================== warnings summary ===============================
[   88s] tests/test_app.py: 35 warnings                                                                  
[   88s]   /home/abuild/rpmbuild/BUILD/kdave-0.2.3/exporter/app.py:240: PendingDeprecationWarning: Function 'semver.compare' is deprecated. Deprecated since version 3.0.0.  Still under investigation, see #258. 
Use the respective 'semver.Version.compare' instead.                                                                                                                                                              
[   88s]     if semver.compare(                                                                          
[   88s]                                                                                                 
[   88s] tests/test_app.py: 14 warnings                                                                  
[   88s]   /home/abuild/rpmbuild/BUILD/kdave-0.2.3/exporter/app.py:270: DeprecationWarning: Function 'semver.parse_version_info' is deprecated. Deprecated since version 2.10.0.  This function will be removed in
 semver 3. Use 'semver.version.Version.parse' instead.                                                   
[   88s]     ver = semver.parse_version_info(parse_semver(version))                                      
[   88s]                                                                                                 
[   88s] -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```
